VideoLocker/src/main/java/org/edx/mobile/util/IOUtils.java

```diff
@@ -0,0 +1,21 @@
+package org.edx.mobile.util;
+
+import android.support.annotation.NonNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.charset.Charset;
+
+import okio.Okio;
+
+public class IOUtils {
+    @NonNull
+    public static String toString(@NonNull InputStream in, @NonNull Charset charset) throws IOException {
+        return Okio.buffer(Okio.source(in)).readString(charset);
```

Comment:
Should we make the last parameter optional? Or maybe even entirely removed. Any files or API responses we read would presumably be be UTF-8 encoded.